doc/openshift-logging-rhocp_4.17+.adoc

```diff
@@ -0,0 +1,114 @@
+logging-loki-ingester-1                        1/1     Running   0          3h44m
+logging-loki-querier-57c8bd8c75-vcc4t          1/1     Running   0          3h45m
+logging-loki-querier-57c8bd8c75-wwlrd          1/1     Running   0          3h45m
+logging-loki-query-frontend-6bbb599859-rmsmk   1/1     Running   0          3h45m
+logging-loki-query-frontend-6bbb599859-zhcfj   1/1     Running   0          3h45m
+cluster-logging-operator-7b6bc9c48-2mjx2       1/1     Running   0          4h22m
+collector-59dxs                                1/1     Running   0          3h16m
+collector-7p6h7                                1/1     Running   0          3h16m
+collector-jbwkn                                1/1     Running   0          3h16m
+collector-ntzm4                                1/1     Running   0          3h16m
+collector-vzlm4                                1/1     Running   0          3h16m
+collector-z75z5                                1/1     Running   0          3h16m
+----
+
+## Tips
+
+You can configure LokiStack and the CustomLogForwarder according to your needs. Check out this link:https://docs.openshift.com/container-platform/4.17/observability/logging/logging-6.0/log6x-loki.html[doc] for how to configure LokiStack storage, and this link:https://docs.openshift.com/container-platform/4.17/observability/logging/logging-6.0/log6x-clf.html[doc] for how to configure log fowarding.
+
+### Object storage
+
+You must have existing object storage in order to configure LokiStack. Loki Operator supports AWS S3, Azure, GCS, Minio, OpenShift Data Foundation, and Swift as options for LokiStack object storage.
+
+Create a secret `logging-loki-s3` inside the `openshift-logging` namespace that contains the required fields needed for LokiStack to access your object storage. As an example, the command below creates a secret that allows access to an OpenShift Data Foundation managed S3 bucket running inside the cluster.
+
+[source,sh]
+----
+  oc create secret generic logging-loki-s3 \
+    --from-literal=access_key_id=key \
+    --from-literal=access_key_secret=secret \
+    --from-literal=bucketnames=bucket-name \
+    --from-literal=endpoint=endpoint \
+    --namespace openshift-logging
+----
+
+### Parsing JSON logs
+
+By default, if container logs are being output in JSON format, they will be nested inside the Vector JSON document's message field. To solve this problem, add the following YAML to the ClusterLogForwarder CR you created earlier:
+
+[source,yaml]
+----
+ kind: ClusterLogForwarder
+ spec:
+   ...
+   filterRefs:
+   - parse-json
+   ...
+   pipelines:
+   - name: default-logstore
+     ...
+     filters:
+     - name: parse-json
+       type: parse
+----
+The addition of the JSON filter above will copy the nested JSON container log into a separate `structured` field inside the Vector JSON document, where the individual fields from the JSON container log can be accessed as `structured. <field_name>`.
```

Comment:
is `structured. <field_name>` a typo?

Reply:
Nope, not a typo, <field_name> is a placeholder for the field names in the liberty log.